src/main/java/net/glowstone/block/blocktype/BlockType.java

```diff
@@ -419,4 +419,23 @@ public void onRedstoneUpdate(GlowBlock block) {
     public void onEntityStep(GlowBlock block, LivingEntity entity) {
         // do nothing
     }
+
+    /**
+     * The rate at which the block should be pulsed.
+     *
+     * @return null if the block should not pulse, or a number of ticks between pulses.
+     */
+    public Integer getPulseTickSpeed(GlowBlock block) {
```

Comment:
I just don't like using `null` as a value most of the time, especially when there is a good alternative.